api/src/pcapi/scripts/algolia_indexing/commands.py

```diff
@@ -57,3 +58,25 @@ def process_venues_from_database(clear: bool, batch_size: int, max_venues: int):
     if clear:
         search.unindex_all_venues()
     batch_indexing_venues_in_algolia_from_database(algolia_batch_size=batch_size, max_venues=max_venues)
+
+
+@blueprint.cli.command("reindex_all_offers")
+@click.option("--max-id", help="max offer id", type=int)
+@click.option("--algolia-batch-size", help="Algolia batch size", type=int, default=1_000)
+@click.option("--max-offers-by-task", help="Maximum number of offers for a cloud task", type=int, default=1_000_000)
```

Comment:
```suggestion
@click.option("--tasks-batch-number", help="Number of batch in a cloud task run", type=int, default=1_000)
```
Je pense que les petits chiffres sont plus simple à gérer.
Ensuite tu peux retrouver max_offers_by_task = algolia_batch_size * tasks_batch_number

Reply:
... d'un autre côté, si on a une grande valeur (plus d'offres à traiter), chaque tâche prendra plus de temps à exécuter. Y a-t-il une durée maximum (timeout) pour les cloud tasks ?

Reply:
Je ne comprends pas ta remarque @dbaty . Avec cette proposition on aura toujours 1_000_000 d'offres gérées par tâche, ce qui change ce sont les paramètres passés.

Reply:
Vu avec Jérémie : j'avais mal lu la suggestion d'Alexis. Mais : 
1. Le nom `tasks-batch-number` me semble peu clair. "batch number", je ne comprends pas ce que ça veut dire. Ça n'est pas le nombre de tâches, en tout cas.
2. Ce nombre de 1_000 est peu clair aussi. C'est 1000 quoi ? Ça n'est pas le nombre d'offres traitées par chaque tâche, en tout cas.

Au final, j'aime bien la proposition originale de Jérémie : `max-offers-by-task` est clair et explicite.